### ISSUE TITLE
1.1.0 Some links documentation don't works

### ISSUE THREAD

**Issue by t1gu1** [flame-engine/flame#1509]
# Link don't exist

When I click on a link I got a 404 error.

# Expected behaviour

Should be redirected to an existing page

# Steps to reproduce

This is a lots of **_here_** links.

In the 1.1.0 Documentation it seems to be only under **flame** category/menu.

---

- Use 1.1.0 Documentation under flame / inputs / Gesture input -> <https://docs.flame-engine.org/1.1.0/flame/inputs/gesture-input.html#example>
- You'll see that text: You can also check more complete examples [here](https://github.com/flame-engine/flame/tree/main/examples/lib/stories/controls/).

The other here link at the end of the same page got the same problem.

---

**Here too:**
- https://docs.flame-engine.org/1.1.0/flame/inputs/keyboard-input.html#receive-keyboard-events-in-a-game-level
- set of the currently pressed [LogicalKeyboardKey](https://api.flutter.dev/flutter/widgets/KeyEventResult-class.html).
- The return value is a [KeyEventResult](https://api.flutter.dev/flutter/widgets/KeyEventResult-class.html).

---

- https://docs.flame-engine.org/1.1.0/flame/inputs/other-inputs.html
- A full examples of how to use it can be found [here](https://github.com/flame-engine/flame/tree/main/examples/lib/stories/input/joystick.dart).

---

- https://docs.flame-engine.org/1.1.0/flame/rendering/images.html#spritebatch
- See the examples [here](https://github.com/flame-engine/flame/tree/main/examples/lib/stories/sprites/spritebatch.dart).

---

There is more but I think you will get it.

**Comment by st-pasha** [comment]
I guess we should add linkcheck as part of the build process...
```
(tutorials/klondike/step3: line   68) ok        http://www.spritecow.com/
(flame/inputs/gesture-input: line   98) ok        https://api.flutter.dev/flutter/gestures/gestures-library.html
(   flame/effects: line  331) ok        https://api.flutter.dev/flutter/animation/Curves-class.html
(flame/inputs/keyboard-input: line   12) ok        https://api.flutter.dev/flutter/widgets/Focus-class.html
(flame/rendering/text: line   38) ok        https://api.flutter.dev/flutter/painting/TextStyle-class.html
(flame/inputs/keyboard-input: line   26) ok        https://api.flutter.dev/flutter/services/RawKeyEvent-class.html
(flame/inputs/gesture-input: line   98) ok        https://api.flutter.dev/flutter/widgets/GestureDetector-class.html
(flame/inputs/keyboard-input: line   96) ok        https://api.flutter.dev/flutter/widgets/FocusNode-class.html
(flame/inputs/keyboard-input: line   26) broken    https://api.flutter.dev/flutter/widgets/KeyEventResult-class.html - 404 Client Error: Not Found for url: https://api.flutter.dev/flutter/widgets/KeyEventResult-class.html
(flame/camera_and_viewport: line    3) ok        https://api.flutter.dev/flutter/widgets/MediaQueryData/devicePixelRatio.html
(flame/inputs/gesture-input: line   98) ok        https://api.flutter.dev/flutter/widgets/RawGestureDetector-class.html
(tutorials/bare_flame_game: line    3) ok        https://code.visualstudio.com/download
(flame/inputs/gesture-input: line   98) ok        https://api.flutter.dev/flutter/widgets/MouseRegion-class.html
(flame/rendering/images: line  185) ok        https://en.wikipedia.org/wiki/Compositing
(tutorials/klondike/klondike: line    3) ok        https://en.wikipedia.org/wiki/Klondike_(solitaire)
(tutorials/klondike/klondike: line    6) ok        https://dart.dev/overview
(   flame/effects: line  541) ok        https://en.wikipedia.org/wiki/Lissajous_curve
(tutorials/klondike/step1: line   30) ok        https://en.wikipedia.org/wiki/Solitaire_terminology
(tutorials/bare_flame_game: line    3) ok        https://docs.flutter.dev/get-started/install
(   flame/effects: line  147) ok        https://en.wikipedia.org/wiki/Tau_(mathematical_constant)
(flame/collision_detection: line  277) broken    https://examples.flame-engine.org/#/Collision%20Detection_Multiple%20shapes - Anchor '/Collision%20Detection_Multiple%20shapes' not found
(   flame/effects: line  564) broken    https://examples.flame-engine.org/#/ - Anchor '/' not found
(flame/collision_detection: line  276) broken    https://examples.flame-engine.org/#/Collision%20Detection_Circles - Anchor '/Collision%20Detection_Circles' not found
(flame/collision_detection: line  278) broken    https://examples.flame-engine.org/#/Collision%20Detection_Shapes%20without%20components - Anchor '/Collision%20Detection_Shapes%20without%20components' not found
(flame/inputs/other-inputs: line  100) broken    https://examples.flame-engine.org/#/Controls_Joystick - Anchor '/Controls_Joystick' not found
(flame/components: line  791) broken    https://examples.flame-engine.org/#/Tile%20Maps_Isometric%20Tile%20Map - Anchor '/Tile%20Maps_Isometric%20Tile%20Map' not found
(tutorials/bare_flame_game: line    3) ok        https://git-scm.com/downloads
(          README: line   72) ok        https://firebase.google.com/
(          README: line   70) ok        https://github.com/Allan-Nava/nakama-flutter
(flame/rendering/palette: line   60) ok        https://github.com/bluefireteam/bgug/blob/master/lib/palette.dart
(tutorials/bare_flame_game: line    3) ok        https://developer.android.com/studio
(flame/rendering/text: line   41) redirect  https://flutter.io/custom-fonts/ - permanently to https://docs.flutter.dev/cookbook/design/fonts
( flame/platforms: line   45) ok        https://github.com/bluefireteam/flutter-gh-pages
(flame/inputs/gesture-input: line  148) ok        https://github.com/flame-engine/flame/blob/main/examples/lib/stories/camera_and_viewport/zoom_example.dart
(          README: line   52) broken    https://github.com/flame-engine/awesome-flame#articles--tutorials - Anchor 'articles--tutorials' not found
(flame/other/debug: line   30) ok        https://github.com/flame-engine/flame/blob/main/examples/lib/stories/components/debug_example.dart
(flame/other/widgets: line    9) redirect  https://github.com/erickzanardo/dashbook - permanently to https://github.com/bluefireteam/dashbook
(flame/components: line  837) redirect  https://github.com/flame-engine/flame/blob/main/examples/lib/stories/effects - permanently to https://github.com/flame-engine/flame/tree/main/examples/lib/stories/effects
(flame/rendering/particles: line  103) ok        https://github.com/flame-engine/flame/blob/main/examples/lib/stories/rendering/particles_example.dart
(      flame/game: line  233) ok        https://github.com/flame-engine/flame/blob/main/examples/lib/stories/system/overlays_example.dart
(flame/components: line  791) broken    https://github.com/flame-engine/flame/blob/main/examples/lib/stories/tile_maps/isometric_tile_map.dart - 404 Client Error: Not Found for url: https://github.com/flame-engine/flame/blob/main/examples/lib/stories/tile_maps/isometric_tile_map.dart
(flame/components: line  810) broken    https://github.com/flame-engine/flame/blob/main/examples/lib/stories/utils/nine_tile_box.dart - 404 Client Error: Not Found for url: https://github.com/flame-engine/flame/blob/main/examples/lib/stories/utils/nine_tile_box.dart
(flame/rendering/particles: line  158) broken    https://github.com/flame-engine/flame/blob/main/examples/lib/stories/utils/particles.dart - 404 Client Error: Not Found for url: https://github.com/flame-engine/flame/blob/main/examples/lib/stories/utils/particles.dart
(          README: line   49) ok        https://github.com/flame-engine/flame/tree/main/examples
(flame/collision_detection: line  279) ok        https://github.com/flame-engine/flame/tree/main/examples/lib/stories/collision_detection
(flame/components: line  826) broken    https://github.com/flame-engine/flame/blob/main/examples/lib/stories/widgets/custom_painter_component.dart - 404 Client Error: Not Found for url: https://github.com/flame-engine/flame/blob/main/examples/lib/stories/widgets/custom_painter_component.dart
(flame/inputs/gesture-input: line  217) broken    https://github.com/flame-engine/flame/tree/main/examples/lib/stories/controls/ - 404 Client Error: Not Found for url: https://github.com/flame-engine/flame/tree/main/examples/lib/stories/controls/
(flame/inputs/gesture-input: line  478) broken    https://github.com/flame-engine/flame/tree/main/examples/lib/stories/input/gesture_hitboxes_example - 404 Client Error: Not Found for url: https://github.com/flame-engine/flame/tree/main/examples/lib/stories/input/gesture_hitboxes_example
(flame/components: line  585) ok        https://github.com/flame-engine/flame/tree/main/examples/lib/stories/parallax
(flame/inputs/keyboard-input: line  105) broken    https://github.com/flame-engine/flame/tree/main/examples/lib/stories/input/keyboard.dart - 404 Client Error: Not Found for url: https://github.com/flame-engine/flame/tree/main/examples/lib/stories/input/keyboard.dart
(flame/inputs/other-inputs: line  100) broken    https://github.com/flame-engine/flame/tree/main/examples/lib/stories/input/joystick.dart - 404 Client Error: Not Found for url: https://github.com/flame-engine/flame/tree/main/examples/lib/stories/input/joystick.dart
(flame/rendering/layers: line  107) broken    https://github.com/flame-engine/flame/tree/main/examples/lib/stories/rendering/layers.dart - 404 Client Error: Not Found for url: https://github.com/flame-engine/flame/tree/main/examples/lib/stories/rendering/layers.dart
(flame/rendering/images: line  180) broken    https://github.com/flame-engine/flame/tree/main/examples/lib/stories/sprites/spritebatch.dart - 404 Client Error: Not Found for url: https://github.com/flame-engine/flame/tree/main/examples/lib/stories/sprites/spritebatch.dart
(flame/rendering/text: line  119) broken    https://github.com/flame-engine/flame/tree/main/examples/lib/stories/rendering/text.dart - 404 Client Error: Not Found for url: https://github.com/flame-engine/flame/tree/main/examples/lib/stories/rendering/text.dart
(flame/rendering/images: line  393) broken    https://github.com/flame-engine/flame/tree/main/examples/lib/stories/sprites/spritesheet.dart - 404 Client Error: Not Found for url: https://github.com/flame-engine/flame/tree/main/examples/lib/stories/sprites/spritesheet.dart
(flame/rendering/images: line  165) broken    https://github.com/flame-engine/flame/tree/main/examples/lib/stories/widget/sprite_widget.dart - 404 Client Error: Not Found for url: https://github.com/flame-engine/flame/tree/main/examples/lib/stories/widget/sprite_widget.dart
(flame/other/widgets: line    9) ok        https://github.com/flame-engine/flame/tree/main/examples/lib/stories/widgets
(flame/rendering/particles: line  158) ok        https://github.com/flame-engine/flame/tree/main/packages/flame/lib/src/particles
(flame/rendering/images: line  316) broken    https://github.com/flame-engine/flame/tree/main/examples/lib/stories/widgets/sprite_animation_widget.dart - 404 Client Error: Not Found for url: https://github.com/flame-engine/flame/tree/main/examples/lib/stories/widgets/sprite_animation_widget.dart
(other_modules/forge2d: line    6) ok        https://github.com/flame-engine/flame/tree/main/packages/flame_forge2d
(other_modules/forge2d: line   11) ok        https://github.com/flame-engine/flame/tree/main/packages/flame_forge2d/example
(other_modules/oxygen: line    5) ok        https://github.com/flame-engine/flame/tree/main/packages/flame_oxygen
(other_modules/oxygen: line   14) ok        https://github.com/flame-engine/flame/tree/main/packages/flame_oxygen/example
(          README: line   46) ok        https://github.com/flame-engine/flame/tree/main/tutorials
(flame_audio/audio: line    5) ok        https://github.com/flame-engine/flame_audio
(flame/components: line  465) broken    https://github.com/flame-engine/flame_flare/tree/main/example - 404 Client Error: Not Found for url: https://github.com/flame-engine/flame_flare/tree/main/example
(flame/components: line  415) ok        https://github.com/flame-engine/flame_flare
(flame/collision_detection: line  228) ok        https://github.com/flame-engine/flame_forge2d
(other_modules/forge2d: line  128) redirect  https://github.com/flame-engine/flame_forge2d/blob/main/example - permanently to https://github.com/flame-engine/flame_forge2d/tree/main/example
(flame/inputs/other-inputs: line  135) ok        https://github.com/flame-engine/flame_gamepad
(other_modules/splash_screen: line   16) ok        https://github.com/flame-engine/flame_splash_screen
(flame/components: line  756) ok        https://github.com/flame-engine/flame_tiled/tree/main/example
(other_modules/tiled: line    5) ok        https://github.com/flame-engine/flame_tiled
(flame/components: line  394) ok        https://github.com/flame-engine/flame_svg
(flame/collision_detection: line  233) ok        https://github.com/flame-engine/forge2d
(other_modules/oxygen: line    5) ok        https://github.com/flame-engine/oxygen
( flame/platforms: line   31) ok        https://pages.github.com/
(other_modules/oxygen: line   11) ok        https://github.com/flame-engine/oxygen/tree/main/doc
(          README: line   38) ok        https://pub.dev/packages/flame/install
(flame/components: line  752) ok        https://github.com/flame-engine/tiled.dart
(flame_audio/audio: line   68) redirect  https://github.com/luanpotter/audioplayer - permanently to https://github.com/bluefireteam/audioplayers
(          README: line   15) ok        https://pub.dev/packages/flame_audio
(          README: line   87) ok        https://pub.dev/packages/dio
(flame_audio/audio: line   13) ok        https://pub.dev/packages/flame_audio/install
(other_modules/oxygen: line   14) ok        https://pub.dev/packages/flame_oxygen
(          README: line   19) ok        https://pub.dev/packages/flame_tiled
(other_modules/splash_screen: line   16) ok        https://pub.dev/packages/flame_splash_screen
(          README: line   17) ok        https://pub.dev/packages/flame_forge2d
(          README: line   86) ok        https://pub.dev/packages/http
(flame/components: line  418) ok        https://pub.dev/packages/flare_flutter
(          README: line   19) ok        https://pub.dev/packages/tiled
(other_modules/splash_screen: line    3) ok        https://raw.githubusercontent.com/flame-engine/flame_splash_screen/main/demogif.gif
(flame/rendering/images: line  321) broken    https://rive.app/#LearnMore - Anchor 'LearnMore' not found
( flame/platforms: line   15) ok        https://skia.org/
(other_modules/tiled: line    3) ok        https://www.mapeditor.org/
build finished with problems.
make: *** [linkcheck] Error 1
```

**Comment by spydon** [comment]
Fixed in #1663
